prow/istio-unit-tests.sh

```diff
@@ -27,9 +27,11 @@ set -e
 source ${ROOT}/prow/lib.sh
 setup_and_export_git_sha
 
+cd ${ROOT}
+
 echo 'Build'
-(cd ${ROOT}; make build)
+make build
 
 # Unit tests are run against a local apiserver and etcd.
 # Integration/e2e tests in the other scripts are run against GKE or real clusters.
-(cd ${ROOT}; make localTestEnv test)
+JUNIT_UNIT_TEST_XML="${ARTIFACTS_DIR}/junit_unit_tests.xml" make localTestEnv test
```

Comment:
Could add "build" before "localTestEnv" and get rid of the separate "make build" invocation done a few lines above.